pages/common/sed.md

```diff
@@ -29,3 +29,7 @@
 - Replace separator / by any other character not used in the find or replace patterns, e.g., #:
 
 `sed 's#{{find}}#{{replace}}#' {{filename}}`
+
+- Print only the `N`th line of a file:
```

Comment:
Let us remove backticks from N, and make it lower case. And to keep it consistent with other examples, use n for the example at line 23 also.